scripts/api/setup-datasetfields.sh

```diff
@@ -7,3 +7,5 @@ curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @da
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/astrophysics.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/biomedical.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/journals.tsv -H "Content-type: text/tab-separated-values"
+curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/computational_workflow.tsv -H "Content-type: text/tab-separated-values"
```

Comment:
**suggestion:** Can we please make this not a default for new installations, instead adding at as an experimental thing?

There is lots of more discussion coming down about how data, code and workflows should be handled and the results might change/replace/alter the approach outlined here. Would be nice to not have new installations add this by default then.

Reply:
I wasn't sure myself when creating the PR so I asked @scolapasta about it. If I recall correctly, he said it was OK to add it in the default setup for new installations as you still need to add it to your collection manually.